drukmakori_sivatag/src/test/java/JUnit/PlayerTest.java

```diff
@@ -1,4 +1,63 @@
 package JUnit;
 
+import drukmakori_sivatag.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
 public class PlayerTest {
```

Comment:
A játék jelenlegi állapotában mindenki szabotálhatja a csöveket, légyszi módosítsátok úgy, hogy erre csak a szabotőrök legyenek képesek. És légyszi módosítsátok az ehhez tartozó teszteket is! :-D

Illetve a Switch Pump teszteset hiányzik, ha jól látom, ha van időtök azt légyszi még csinájátok meg!